### DISCUSSION TITLE
tutorials/intro-hotchocolate/11-playlist-tracks

### DISCUSSION THREAD

**Discussion by giscus** [apollographql-education/content-feedback#110] [course feedback]
# tutorials/intro-hotchocolate/11-playlist-tracks

Learn how to connect object types in your schema.

https://www.apollographql.com/tutorials/intro-hotchocolate/11-playlist-tracks

**Comment by MatthewMucker** [comment]
The file under the "See the full track.cs file" does show track.cs, but the header line when the section is expanded says Playlist.cs instead of Track.cs.

**Reply by mabuyo** [reply]
Good catch, thanks! We'll fix this shortly.

**Comment by MatthewMucker** [comment]
After adding the Tracks field to the Playlist class, the compiler throws a warning that the two constructors not touched in this lesson don't return a Tracks list. You probably want to add a line 'Tracks = new List<Track>();' in each of those two constructors. (Either that or make the Tracks field nullable, maybe...)

**Reply by mabuyo** [reply]
Yes you're correct! However, by the end of the course, these compiler warnings will disappear.
